python-client/.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+# ref: https://docs.gitlab.com/ee/ci/README.html
+
+stages:
```

Comment:
why do we need gitlab ci?

Reply:
no, its output from the generator. Will remove it in the action. Also travic.ci and git-push.sh are now removed since they are not used.